model/countries/BR/BR-formatting-rules.yaml

```diff
@@ -104,7 +101,6 @@ examples:
       Clube dos Preenchedores Automáticos
```

Comment:
This needs to change - also in several other places.

If you open the generated HTML files you see the `<p class="formatting_errors">` blocks. They should go away.

Reply:
Ack. Should be ok now. PTAL.